target/linux/ipq40xx/base-files/lib/preinit/05_set_iface_mac_ipq40xx.sh

```diff
@@ -37,7 +37,8 @@ preinit_set_mac_address() {
 		ip link set dev sw-eth1 address "$base_mac"
 		ip link set dev sw-eth2 address $(macaddr_add "$base_mac" 1)
 		;;
-	zyxel,nbg6617)
+	zyxel,nbg6617|\
+	zyxel,wsq50)
```

Comment:
This should be done via NVMEM, then there is no need for this

Reply:
Migrated to NVMEM and confirmed working